src/win/winapi.c

```diff
@@ -59,6 +62,12 @@ void uv_winapi_init(void) {
 
   pRtlGetVersion = (sRtlGetVersion) GetProcAddress(ntdll_module,
                                                    "RtlGetVersion");
+  if (pRtlGetVersion != NULL) {
+    memset(&info, 0, sizeof(info));
+    info.dwOSVersionInfoSize = sizeof(info);
+    pRtlGetVersion(&info);
+    is_windows_10_or_greater = (info.dwMajorVersion >= 10);
```

Comment:
Do we need a fallback setting for `is_windows_10_or_greater` when `pRtlGetVersion  == NULL`?

Reply:
No, because in that case `is_windows_10_or_greater == 0` because it's in static storage (variables in bss are zero-initialized.)